src/addReviewServer.js

```diff
@@ -0,0 +1,17 @@
+import fetch from "node-fetch";
+
+const API_BASE = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+
+export async function postReview(req, res) {
+  const method = {
+    method: "POST",
+    body: JSON.stringify(req.body),
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+  };
+  // i am going to use this variable in the second part of the assignment.
+  const response = await fetch(API_BASE, method);
+  res.send().status(200);
```

Comment:
I think the HTTP status 201 is better suited for when you've created something, like here when you've created a review.